### PR TITLE
chore: user guide updates

### PR DESCRIPTION
**Pull request by sr-remsha**

### CODE REVIEW

File: docs/user-guide.md [hunk 1]
```diff
@@ -54,41 +56,59 @@ This is the list of all the supported actions. **Note**, that actions may vary f
...
 The following symbols in the folders names are prohibited: tab, `"`, `:`, `;`, `/`, `\`, `,`, `=`, `{`, `}`, `%`, `&` and will be excluded. Note that you can use the `.` symbol at the start or inside a name, but the dot at the end will be automatically removed.
 
 The MAX length of the folder name is limited to 160 symbols. Everything beyond is cut off. 
 
-> When you use the contextual menu to move a conversation, you can create a new folder, in which the conversation should be moved.
-
 ### Search and Filter
 
-The **Search** box can be utilized to find conversations and folders by their names.
+The **Search** box can be utilized to find conversations and folders by their names. If you have any share d conversations, you can apply **Shared by me** filter to sort them out.
```
Comment (VolhaBazhkova) [inline docs/user-guide.md:83]: typo in "share d"

File: docs/user-guide.md [hunk 2]
```diff
@@ -98,56 +118,73 @@ In the conversation header, you can view the current settings, change settings (
...
+
+To share a conversation or a folder, click **Share** in the contextual menu and copy the link.
 
 ![](./img/share_conv.png)
 
+After the recipient has opened your link, the arrow icon appears near its name.
+
+![](./img/shared.png)
+
+Click **Shared by me** checkbox to select just the conversations that you have shared:
```
Comment (VolhaBazhkova) [inline docs/user-guide.md:143]: checkbox? maybe filter or checkbox in the filter?

File: docs/user-guide.md [hunk 3]
```diff
@@ -98,56 +118,73 @@ In the conversation header, you can view the current settings, change settings (
...
+**To share a conversation**:
 
-When you share a folder, all folders and conversations in this folders will be shared.
+> **Note**: You can share entire folders with conversations. When you share a folder, all folders and conversations in it will be shared as well.
+
+To share a conversation or a folder, click **Share** in the contextual menu and copy the link.
 
 ![](./img/share_conv.png)
 
+After the recipient has opened your link, the arrow icon appears near its name.
```
Comment (VolhaBazhkova) [inline docs/user-guide.md:139]: Actually user has to re-login or refresh the browser to see the arrow for shared chat/prompt. Do two conditions: to click the link by user2 and refresh the page by user1

File: docs/user-guide.md [hunk 4]
```diff
@@ -160,13 +197,19 @@ To export all conversations, at the bottom of the left panel, click the **Export
...
 
-To import JSON or a zip file with conversations, click the **Import conversations** icon at the bottom of the left panel and then select a file with conversations.
+To import a JSON file with a conversation or a ZIP archive with several conversations (may include attachments as well), click the **Import conversations** icon at the bottom of the left panel and then select a file with conversations.
 
-> When you import a conversation with attachments, the attachments will be available in the Imports folder.
- 
 ![](./img/import_conv.png)
 
-> Note, that when importing a duplicate of an existing conversation, you will be prompted to select one of the proceeding options: replace, ignore or add a prefix to the imported conversation. If the imported conversation includes attachments, you will be prompted to make the same selection for them as well.
+When you import a conversation with attachments, the attachments will be available in the **Imports** folder in the [Attachments Manager](#attachments-manager).
```
Comment (VolhaBazhkova) [inline docs/user-guide.md:204]: Sorry, this has been changed. Now there is no specific folder created after import. The attachments will be available in the parent folder or in the root folder. We removed creation of import folder.

File: docs/user-guide.md (hunk 1 above)
Comment (sr-remsha) [reply docs/user-guide.md]: fixed

File: docs/user-guide.md [hunk 5]
```diff
@@ -54,41 +56,59 @@ This is the list of all the supported actions. **Note**, that actions may vary f
...
+When you create a new conversation, it is automatically named after your first prompt. After that, you can rename your conversation.
+
+**To rename a conversation**:
+
+1. Click **Rename** in the context menu of the selected conversation.
+2. Enter a new name and submit.
+
+![](./img/rename-conversation.png)
+
+The following symbols in the conversation names are prohibited: tab, `"`, `:`, `;`, `/`, `\`, `,`, `=`, `{`, `}`, `%`, `&` and will be excluded. Note that you can use the `.` symbol at the start or inside a name, but the dot at the end will be automatically removed. The MAX length of the conversation name is limited to 160 symbols. Everything beyond is cut off.
```
[the remark this answers is not in the document]
Comment (sr-remsha) [reply docs/user-guide.md:111]: added

File: docs/user-guide.md (hunk 4 above)
Comment (sr-remsha) [reply docs/user-guide.md:204]: fixed

File: docs/user-guide.md (hunk 2 above)
Comment (sr-remsha) [reply docs/user-guide.md]: fixed

**Review by VolhaBazhkova — APPROVED** [review]

**Pull request merged**